aemedge/blocks/global-banner/global-banner.css

```diff
@@ -72,6 +72,13 @@ main .block.global-banner[data-block-status="loaded"].cyan-black a{
   z-index: 11;
 }
 
+.static span.close {display: none}
+
+main .block.global-banner[data-block-status="loaded"].static {
+  justify-content: center;
+  margin-left: -40px;
```

Comment:
Can you try leaving this negative margin off? I don't think you need it, and it is making content get cut off in mobile. 
![image](https://github.com/user-attachments/assets/847a7dc8-4026-4a07-9c49-f9dcff95b4e8)